src/main.tsx

```diff
@@ -6,6 +6,7 @@ import "./global.css";
 import { NavbarScreen } from "./components/Nav/Nav.tsx";
 import { Login } from "./screens/Login.tsx";
 import { Register } from "./screens/Register.tsx";
+// import { UpdatePassword } from "./components/UpdatePassword/UpdatePassword.tsx";
```

Comment:
The **component** is called as such from the `screen` folder, not from the main file.